### PR TITLE
Fix: Inbox filters

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: []

## What?
Fix issue with numbers of Markets in intermediateMarketFilterMap

## Why?
-

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/portals/hope/features/filters/hooks/use-selected-filters.ts [hunk 1]
```diff
@@ -15,15 +15,17 @@ export const useSelectedFilters = () => {
...
-    6: Market.Denmark,
+    [memberGroupsLength]: Market.Sweden,
+    [memberGroupsLength + 1]: Market.Norway,
+    [memberGroupsLength + 2]: Market.Denmark,
   }
 
   const applyIntermediateFilter = (filters: number[]) =>
     filters.filter((filter) => {
-      if (filter >= 4 && filter <= 6) {
+      if (filter >= 6 && filter <= 8) {
```
Comment (vonElfvin) [inline src/portals/hope/features/filters/hooks/use-selected-filters.ts:28]: Any way we can avoid these hard coded numbers as well?

**Review by vonElfvin — APPROVED** [review]
🔥

File: src/portals/hope/features/filters/hooks/use-selected-filters.ts (hunk 1 above)
Comment (vonElfvin) [reply src/portals/hope/features/filters/hooks/use-selected-filters.ts:28]:
```suggestion
      if (memberGroupsLength >= 6 && filter <= memberGroupsLength +2) {
```

Comment (maxplt) [reply src/portals/hope/features/filters/hooks/use-selected-filters.ts:28]: Oh, yeah, missed them

**Pull request merged**